components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultLogoutRequestHandler.java

```diff
@@ -537,11 +542,8 @@ private boolean isValidCallerPath(AuthenticationContext context) {
             } catch (IdentityApplicationManagementException e) {
                 return false;
             }
-        } else {
-            String enableValidatingLogoutReturnUrl = IdentityUtil.getProperty
-                    (ENABLE_VALIDATING_LOGOUT_RETURN_URL_CONFIG);
-            return !Boolean.valueOf(enableValidatingLogoutReturnUrl);
         }
+        return Boolean.FALSE;
```

Comment:
add a debug point mentioning that context.getRelyingParty() is empty hence can't validate the logout return url.